typed-racket-lib/typed-racket/static-contracts/combinators/control.rkt

```diff
@@ -52,16 +50,11 @@
 
 
 (define (pt-seq-map f seq)
-  (match seq
-   [(pt-seq vals call-cc)
-    (define (f* a) (f a 'invariant))
-    (pt-seq
-      (map f* vals)
-      (and call-cc (map f* call-cc)))]))
+  (match-define (pt-seq vals call-cc) seq)
+  (define (f* a)
```

Comment:
I really want a way to turn this formatting decision off.

Reply:
This function in particular hardly seems worth defining at all.